src/taipy/gui/data/decimator/lttb.py

```diff
@@ -29,7 +29,7 @@ def _areas_of_triangles(a, bs, c):
         a_minus_bs = a - bs
         return 0.5 * abs((a[0] - c[0]) * (bs_minus_a[:, 1]) - (a_minus_bs[:, 0]) * (c[1] - a[1]))
 
-    def decimate(self, data: np.ndarray) -> np.ndarray:
+    def decimate(self, data: np.ndarray, payload: t.Any) -> np.ndarray:
```

Comment:
Shoudn't payload's type be a bit more precise ? t.Dict[str, t.Any] ?

Reply:
thanks!!